arrays homework .playground/Contents.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+var greeting = "Hello, playground"
+var  mycity = "Medina"
+var cities = [String]()
+cities.append("Medina")
+cities.append("jeddah")
+cities.append("Makkah")
+cities.append("Riyadh")
+cities.append("Taif")
+cities.append("Hail")
+cities.append("Khobar")
+cities.append("Dammam")
+cities.append("Qassim")
+cities.append("Jazan")
+
+var pop = [Int]()
+pop.append(1000)
+pop.append(2000)
+pop.append(3000)
+pop.append(4000)
+pop.append(5000)
+pop.append(6000)
+pop.append(7000)
+pop.append(8000)
+pop.append(9000)
+pop.append(10000)
+
+for I in cities {
+    print (I)
+}
+
+var L = 5000
+switch L {
+case 1000 , 2000 , 3000 , 4000 , 5000 :
+    print(" it is a small city")
+case 6000 , 7000 , 8000 , 9000 , 10000 :
+    print (" it is a big city")
+
+default :
+print ("it is " )
+}
+
+if (mycity == cities[0]){
+ print(pop[0])
+}
+else if (mycity == cities [1]){
+    print(pop[1])}
+else if (mycity == cities[2]){
+    print(pop[2])
+}
+if (mycity == cities [3]){
```

Comment:
```suggestion
else if (mycity == cities [3]){
```
Please apply 'else if' for all cities, then at the end you could have
`else { print("Selected city not found") }`